frontend/views/containers/access/PasswordForm.vue

```diff
@@ -67,15 +97,95 @@ export default ({
       required: false
     }
   },
+  methods: {
+    generateRandomPassword (pwLen = 32) {
```

Comment:
Based on the crypto primitives we use for encryption, I think 128 bits (i.e., 32 chars long passwords, assuming hex) is a reasonable value.